### ISSUE TITLE
Unable to build docker-volume-glusterfs on my mac

### ISSUE THREAD

**Issue by ksachdev1** [docker/docker#31312]
**Description**
unbale to install glusterfs driver on Mac OS 10.11.6 Docker version 1.13.0

I have a glusterfs cluster running as a cluster of 2 nodes on CentOS 7. I am trying to access that using my mac os x 10.11.6 which has Docker version 1.13.0, build 49bf474 installed on it.

I have been using the below link to install the driver for dockers to get going but I keep getting build error. This is the error I get when i use "go get..."

glusterplugin]go get github.com/calavera/docker-volume-glusterfs

github.com/calavera/docker-volume-glusterfs

../../../../go/src/github.com/calavera/docker-volume-glusterfs/main.go:38: cannot use d (type glusterfsDriver) as type volume.Driver in argument to volume.NewHandler: glusterfsDriver does not implement volume.Driver (wrong type for Mount method) have Mount(volume.Request) volume.Response want Mount(volume.MountRequest) volume.Response ../../../../go/src/github.com/calavera/docker-volume-glusterfs/main.go:39: cannot use "glusterfs" (type string) as type int in argument to h.Handler.ServeUnix

Basically, I need the docker-volume-glusterfs binary so that I can configure the gluster cluster info on my machine and then start a docker container which can use that gluster cluster. The build itself keeps failing to build the binary "docker-volume-glusterfs"
**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
(paste your output here)
```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by cpuguy83** [comment]
This should be dealt with in the relevant repo.
There is nothing we can do here to fix the build issue.

Closing since this is not a docker issue.

You will either need to change the driver code to I use the updated go-plugin-helpers and go-connections or downgrade these packages in your local gopath.

**Comment by ksachdev1** [comment]
I tried finding the relevant repo, could you point me on whom to reach out to for this? Is there any suggestion you would have. I am trying to understand the last part of your msg:"You will either need to change the driver code to I use the updated go-plugin-helpers and go-connections or downgrade these packages in your local gopath"

**Comment by thaJeztah** [comment]
@ksachdev1 repo is here; https://github.com/calavera/docker-volume-glusterfs
